src/lib/modules/Patron/PatronPastLoans/actions.js

```diff
@@ -6,23 +6,27 @@ export const IS_LOADING = 'fetchPatronPastLoans/IS_LOADING';
 export const SUCCESS = 'fetchPatronPastLoans/SUCCESS';
 export const HAS_ERROR = 'fetchPatronPastLoans/HAS_ERROR';
 
-const selectQuery = (patronPid, page) => {
+const selectQuery = (patronPid, page, size) => {
   return loanApi
     .query()
     .withPatronPid(patronPid)
     .withState(invenioConfig.circulation.loanCompletedStates)
     .withPage(page)
+    .withSize(size)
```

Comment:
@topless this is the fix that you were mentioning this morning about the pagination